graphics/macgui/macdialog.cpp

```diff
@@ -116,9 +119,9 @@ void Dialog::paint() {
 
 		if ((int)i == _pressedButton && _mouseOverPressedButton) {
 			Common::Rect bb(button->bounds.left + 5, button->bounds.top + 5,
-				button->bounds.right - 5, button->bounds.bottom - 5);
+							button->bounds.right - 5, button->bounds.bottom - 5);
 
-			Design::drawFilledRect(&_gui->_screen, bb, kColorBlack, _gui->_wm->getPatterns(), kPatternSolid);
+			Wage::Design::drawFilledRect(&_gui->_screen, bb, kColorBlack, _gui->_wm->getPatterns(), kPatternSolid);
```

Comment:
Err, how Graphics:: could call a specific engine? Please refactor, the common code cannot refer to any engine